vueui/src/components/view/admin/permissions-panel.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <div v-if="forumuserui_enabled" class="notice">
+      <center>Modifications done do not outlast the round!</center>
+    </div>
+    <div class="table">
+      <div class="header">
+        <div class="header-item">Ckey <vui-button :params="{ action: 'add' }">Add</vui-button></div>
+        <div class="header-item">Rank</div>
+        <div class="header-item">Permissions</div>
+      </div>
+
+      <div class="entry" v-for="admin in admins" :key="admin.ckey">
+        <div class="item">
+          <vui-button :params="{ action: 'remove', ckey: admin.ckey }">Edit</vui-button> {{admin.ckey}}
+        </div>
+        <div class="item">
+          <vui-button :params="{ action: 'rank', ckey: admin.ckey }">Edit</vui-button> {{admin.rank}}
+        </div>
+        <div class="item" width="30%">
+          <vui-button :params="{ action: 'rights', ckey: admin.ckey }">Edit</vui-button> {{admin.rights}}
+        </div>
+      </div>
```

Comment:
Use `<table>`, `<tr>`, `<th>` and  `<td>` instead of divs using table display.

Reply:
:thinking: this was copied from your player panel tho.